osu.Game/Overlays/Changelog/ChangelogSupporterPromo.cs

```diff
@@ -0,0 +1,154 @@
+                            Children = new Drawable[]
+                            {
+                                new FillFlowContainer
+                                {
+                                    RelativeSizeAxes = Axes.X,
+                                    AutoSizeAxes = Axes.Y,
+                                    Direction = FillDirection.Vertical,
+                                    Anchor = Anchor.CentreLeft,
+                                    Origin = Anchor.CentreLeft,
+                                    Padding = new MarginPadding { Right = 50 + image_container_width },
+                                    Children = new Drawable[]
+                                    {
+                                        new OsuSpriteText
+                                        {
+                                            Text = ChangelogStrings.SupportHeading,
+                                            Font = OsuFont.GetFont(size: 20, weight: FontWeight.Light),
+                                            Margin = new MarginPadding { Bottom = 20 },
+                                        },
+                                        supportLinkText = new LinkFlowContainer(t =>
+                                        {
+                                            t.Font = t.Font.With(size: 14);
+                                        })
+                                        {
+                                            RelativeSizeAxes = Axes.X,
+                                            AutoSizeAxes = Axes.Y,
+                                        },
+                                        supportNoteText = new TextFlowContainer(t =>
+                                        {
+                                            t.Font = t.Font.With(size: 12);
+                                        })
+                                        {
+                                            Margin = new MarginPadding { Top = 10 },
+                                            RelativeSizeAxes = Axes.X,
+                                            AutoSizeAxes = Axes.Y,
+                                        }
+                                    },
+                                },
+                                imageContainer = new Container
+                                {
+                                    RelativeSizeAxes = Axes.Y,
+                                    Width = image_container_width,
+                                    Anchor = Anchor.CentreRight,
+                                    Origin = Anchor.CentreRight,
+                                }
+                            }
+                        },
+                    }
+                },
+            };
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colour, TextureStore textures)
+        {
+            void fontPinkColour(SpriteText t) => t.Colour = colour.PinkLighter;
```

Comment:
`LinkFlowContainer`s accept default sprite text creation parameters via ctor, so this could be moved there.